test/source/tests/compose.ts

```diff
@@ -821,7 +820,7 @@ export const defineComposeTests = (testVariant: TestVariant, testWithBrowser: Te
       const fileText = await composePage.awaitDownloadTriggeredByClicking(async () => {
         await attachment.click('#download');
       });
-      expect(fileText.toString()).to.equal(`small text file\nnot much here\nthis worked\n`);
+      expect(Object.values(fileText).pop()!.toString()).to.equal(`small text file\nnot much here\nthis worked\n`);
       await composePage.close();
```

Comment:
For all calls to `awaitDownloadTriggeredByClicking` in tests, we should start checking for the exact filename. Also need to rename `fileText` to `downloadedFiles` or similar.

Something like

```diff
- expect(Object.values(fileText).pop()!.toString()).to.equal(`small text file\nnot much here\nthis worked\n`);
+ expect(downloadedFiles["some-specific-file-name.txt"].toString()).to.equal('tested content')
```

and so on.